Lesson1/T_Ilya/CPP_ADV_Lesson1/task2.cpp

```diff
@@ -0,0 +1,56 @@
+#include "task.h"
+#include "functions.h"
+
+
+
+namespace MySpace{
+int task2() // -----LIST-----
+{
+    int x{10};
+    int y{5};
+    int a{19};
+    int b{4};
+    std::vector<int> vect_1{};
+    vect_1.reserve(5);
```

Comment:
use corresponding constructor of vector instead of reserve.

Reply:
done or not???